verify.sh

```diff
@@ -41,6 +41,8 @@ function verify_libbpf() {
         echo "no libbpf package found"
         exit 1
     fi
+    sudo apt-get install -y mlocate
+    locate libbpf
```

Comment:
My preference would be to rely on pkgconfig to check the all things are installed. 
```
pkg-config --modversion libbpf
```
This is already installed and IMHO, should be used to obtain the `GO_CFLAGS` and `--static` library

Reply:
https://github.com/sustainable-computing-io/kepler-action/actions/runs/7059814600/job/19218145143?pr=89#step:4:16 @sthaha , could you please try `pkg-config` in another PR?